clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaStoreIngestionService.java

```diff
@@ -1457,16 +1465,24 @@ public String prepareIngestionInfoFor(String storeName, Integer version, Integer
         ingestingTopic = pubSubTopicRepository.getTopic(Utils.composeRealTimeTopic(storeName));
       }
       PubSubTopicPartition ingestingTopicPartition = new PubSubTopicPartitionImpl(ingestingTopic, partition);
-      return infoPrefix
-          + aggKafkaConsumerService.getIngestionInfoFor(versionTopic, ingestingTopicPartition, regionName);
+
+      String consumerServiceIngestionInfo =
+          aggKafkaConsumerService.getIngestionInfoFor(versionTopic, ingestingTopicPartition, regionName);
+      if (consumerServiceIngestionInfo == null || consumerServiceIngestionInfo.isEmpty()) {
+        // Skip logging if no info is available, mainly due to print to frequently for that consumer
+        return;
+      }
+      LOGGER.warn(
```

Comment:
Let's put this behind debug flag

Reply:
The point is we do not know the heartbeat alert happen, especially during midnight, original solution is using admin-tool to fetch this info from server, but human often missed that.  So we add logic here to print out logs. If we add debug, that means we fall back to the original solution to require human involving this.